fenix/app/src/androidTest/java/org/mozilla/fenix/ui/NimbusMessageHomescreenTest.kt

```diff
@@ -0,0 +1,119 @@
+package org.mozilla.fenix.ui
```

Comment:
nit: missing license.

Reply:
I am here to stay, so that would be nice, no rush though.